ynput/OpenPype
# Add new action to create project structure only on ftrack

## Changelog Description
Add a new action to create Ftrack hierarchy and avoid folder structure creation. 

## Additional info
By now, we are only able to create folder structure and also Ftrack hierarchy if needed. The problem is for some departments, we just require the structure on Ftrack, and we don't want any folder creation. 

## Testing notes:
1. Create a folder structure with [Ftrack] identifier for the project.
2. Run the Create Ftrack Project Structure action on Ftrack.
3. Just the Ftrack part of the structure should be created and only on Ftrack.
